blosc2/SChunk.py

```diff
@@ -104,7 +104,8 @@ def __init__(self, chunksize=None, data=None, **kwargs):
         >>> schunk = blosc2.SChunk(**storage)
         """
         # Check only allowed kwarg are passed
-        allowed_kwargs = all(kwarg in ["urlpath", "contiguous", "cparams", "dparams", "schunk", "mode"]
+        allowed_kwargs = all(kwarg in ["urlpath", "contiguous", "cparams", "dparams", "schunk",
+                                       "mode", "_is_view"]
                              for kwarg in kwargs.keys())
         if not allowed_kwargs:
             raise ValueError("Only `urlpath`, `contiguous`, `cparams`, `dparams` and `mode` "
```

Comment:
`schunk` is not among the allowed kwargs.  Is that intended?  It would not be better to isolate the not allowed keyword and raise `f"Not allowed kwarg : {not_allowed_kwarg}"`?

Reply:
Fixed in 65dcb6bf8dad3deae5d0c0fcb5cf9b95ec17da3f.